source/Audio.cpp

```diff
@@ -228,6 +247,24 @@ void Audio::SetVolume(double level)
 
 
 
+// Get the volume.
+double Audio::MusicVolume()
+{
+	return musicVolume;
+}
+
+
+
+// Set the volume (to a value between 0 and 1).
+void Audio::SetMusicVolume(double level)
+{
+	musicVolume = min(1., max(0., level));
```

Comment:
now that we have C++ 17 support, we can use `std::clamp`
```suggestion
	musicVolume = clamp(level, 0., 1.);
```